frontend/src/components/admin/EmptyTable.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+
+import { Button, Flex, Heading, Text, Tr, Td } from "@chakra-ui/react";
+
+export type EmptyTableProps = {
+  filters: { (data: string | null): void }[];
+};
+
+const EmptyTable = ({ filters }: EmptyTableProps) => {
+  const clearFilters = () => {
+    filters.forEach((f) => {
```

Comment:
nit: `f->setFilter` this gave me a little heart attack it thought you were directly setting state LOL